data/json/monster_special_attacks/monster_gun.json

```
@@ -269,6 +269,28 @@
     ],
     "melee_damage": { "bash": 8 }
   },
+  {
+    "id": "civilian_glock19",
+    "copy-from": "glock_19",
+    "looks_like": "glock_19",
+    "flags": [ "PSEUDO" ],
+    "type": "GUN",
+    "name": { "str_sp": "Fighters' Glock 19" },
+    "description": "A gun carried by civilian monsters, more inaccurate for balancing reasons.",
```

```suggestion
    "description": { "str": "A gun carried by civilian monsters, more inaccurate for balancing reasons.  If you see this, it's a bug.", "//~": "NO_I18N" },
```
This prevents stuff like this from being translated, should do the same with the names if they don't show up in game either. (Alternately just don't change the copy-frommed names and descriptions and instead add the reasoning in a "//" comment)